opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -54,6 +54,12 @@
                 "description": "The docker image and tag for running the OpenCGA service. For examples: 'repo/image:tag'"
             }
         },
+        "batchContainerImage": {

```

Comment:
Does this need adding to the `azuredeploy.parameters.json`

Reply:
Done.